samples/durable-task-sdks/dotnet/AspNetWebApp/README.md

```diff
@@ -205,7 +205,7 @@ Once logged in, you should see the orchestrations that were created by the sampl
 
 ## Optional: Deploy to Azure Container Apps
 
-1. Create an container app following the instructions in the [Azure Container App documentation](https://learn.microsoft.com/azure/container-apps/get-started?tabs=bash).
+1. Create an container app following the instructions in the [Azure Container App documentation](https://learn.microsoft.com/en-us/azure/container-apps/quickstart-code-to-cloud?tabs=bash%2Ccsharp).
```

Comment:
```suggestion
1. Create an container app following the instructions in the [Azure Container App documentation](https://learn.microsoft.com/azure/container-apps/quickstart-code-to-cloud?tabs=bash%2Ccsharp).
```

so region and language aren't set automatically (more friendly for users in other regions)